### PR TITLE
feat(textarea): textarea manual testing

### PR DESCRIPTION
**Pull request by nathalielindqvist**

### CODE REVIEW

File: src/pages/FormPage.tsx [hunk 1]
```diff
@@ -232,12 +260,21 @@ const FormPage = () => {
...
               <tds-textarea
                 mode-variant="secondary"
-                name="textarea"
-                label="What do you do at Scania?"
+                name="occupationalEssay"
+                label="What do you do at Scania? (Minimum 50 chars)"
                 label-position="outside"
+                state={occupationEssayState}
+                helper={helperTextOccupationEssayState}
+                disabled={textareaDisabled === true}
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:272]:
```suggestion
                disabled={textareaDisabled}
```
The === true isn't needed here

File: src/pages/FormPage.tsx [hunk 2]
```diff
@@ -45,6 +49,13 @@ const FormPage = () => {
     }
   }, [countrySelected]);
 
+  useEffect(() => {
+    if(addressValue !== null && addressValue !== "") {
+      setTextareaDisabled(false)
+    }
+    
+  }, [addressValue]);
+
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:58]:
```suggestion
  useEffect(() => {
      setTextareaDisabled(addressValue !== null && addressValue !== "")
  }, [addressValue]);

```

Need to reset it to disabled if the user first types in an adress and then removes it. The suggestion above should work

File: src/pages/FormPage.tsx [hunk 3]
```diff
@@ -132,6 +151,13 @@ const FormPage = () => {
                 no-min-width
                 placeholder="Majorvägen 32"
                 helper={addressValidation ? "" : "Address is mandatory field!"}
+                ref={(element) => {
+                  element?.addEventListener('tdsInput', (event) => {
+                    const customEvent = event as CustomEvent;
+                    const { value } = customEvent.detail.target;
+                    setAddressValue(value)
+                  })
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:159]:
```suggestion
                  element?.addEventListener('tdsInput', (event as CustomEvent) => {
                    setAddressValue(event.detail.target)
                  })
```

Not sure if this works but would remove a couple of lines of code

**Review by alexandereneroth — APPROVED** [review]
nice! 
Left some feedback

File: src/pages/FormPage.tsx [hunk 4]
```diff
@@ -45,6 +49,11 @@ const FormPage = () => {
     }
   }, [countrySelected]);
 
+  useEffect(() => {
+    setTextareaDisabled(addressValue === null || addressValue === "")
```
[the remark this answers is not in the document]
Comment (nathalielindqvist) [reply src/pages/FormPage.tsx]: Awesome, thank you! Updated 😄

File: src/pages/FormPage.tsx [hunk 5]
```diff
@@ -54,20 +63,30 @@ const FormPage = () => {
         setAddressValidation(false);
       } else {
         setAddressValidation(true);
-        formData.forEach((value, key) => {
-          console.log("Key:", key, "Value:", value);
-        });
+        const occupationalEssay = formData.get("occupationalEssay");
+
+        if (occupationalEssay !== undefined && occupationalEssay && occupationalEssay.toString().length < 50) {
```
[the remark this answers is not in the document]
Comment (nathalielindqvist) [reply src/pages/FormPage.tsx]: Love the feedback! Updated 🥳

**Pull request merged**